jetty-websocket/websocket-core-common/src/main/java/org/eclipse/jetty/websocket/core/internal/messages/InputStreamMessageSink.java

```diff
@@ -17,12 +17,19 @@
 
 import org.eclipse.jetty.websocket.core.CoreSession;
 import org.eclipse.jetty.websocket.core.Frame;
+import org.eclipse.jetty.websocket.core.internal.util.MethodHolder;
 
 public class InputStreamMessageSink extends DispatchedMessageSink
 {
+    public InputStreamMessageSink(CoreSession session, MethodHolder methodHolder)
+    {
+        super(session, methodHolder);
+    }
+
+    @Deprecated
     public InputStreamMessageSink(CoreSession session, MethodHandle methodHandle)
```

Comment:
Why did you keep this constructor, while in all other `Sink`s it was removed (to only have the `MethodHolder` constructor)?
It's an internal class so we can break compatibility.